capstone/src/main/java/com/google/univiz/api/CollegeId.java

```diff
@@ -0,0 +1,7 @@
+package com.google.univiz.api;
+
+/**
+ * The CollegeId class will contain a college name and a unique college ID that can be used to
+ * identify the college
+ */
+public class CollegeId {}
```

Comment:
I have a feeling this would later become an AutoValue and that's why you didn't make it `final. But if this is not the case then mark it as `final`.

```suggestion
public final class CollegeId {}
```

Reply:
I do believe this will eventually become Autovalue :)